.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "13:00"
```

Comment:
Is a time mandatory? Can we just leave it out?

Reply:
I don't know, that's what we usually configure for other vox pupuli projects.